src/interceptors/fresh-firefox.ts

```diff
@@ -41,13 +40,13 @@ const testCertutil = (command: string, options?: SpawnOptions) => {
 
 const getCertutilCommand = _.memoize(async () => {
     // If a working certutil is available in our path, we're all good
-    if (await testCertutil('certutil')) return { command: 'certutil' };
+    if (await testCertutil('certutil')) return {command: 'certutil'};
```

Comment:
Can you undo all the formatting changes from this file? It makes this quite difficult to review, and it breaks `git blame` later on.

Reply:
oh sorry, yea sure